frontend/src/utils/fieldValidation.ts

```diff
@@ -431,12 +431,11 @@ export const createDateValidationRules: ValidationRuleFn<DateFieldBase> = (
         }
 
         const { customMinDate, customMaxDate } = schema.dateValidation ?? {}
+        const customMinNoTime = loadDateFromNormalizedDate(customMinDate)
+        const customMaxNoTime = loadDateFromNormalizedDate(customMaxDate)
         return (
-          !isDateOutOfRange(
-            parseDate(val),
-            fromUtcToLocalDate(customMinDate),
-            fromUtcToLocalDate(customMaxDate),
-          ) || 'Selected date is not within the allowed date range'
+          !isDateOutOfRange(parseDate(val), customMinNoTime, customMaxNoTime) ||
```

Comment:
nit: realised you refactored `loadDateFromNormalizedDate`, Then can just refactor `customMinNoTime, customMaxNoTime` too as its only used once, and the naming is a bit weird too.

```
!isDateOutofRange(parseDate(val),
    loadDateFromNormalizedDate(customMinDate),
    loadDateFromNormalizedDate(customMaxDate),
    ) || 'Selected date is not within the allowed date range'
```
Same for `DateField.tsx`

Reply:
done in ed2b31fd